pkg/crc/image/image.go

```diff
@@ -0,0 +1,129 @@
+package image
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/url"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/code-ready/crc/pkg/crc/logging"
+	crcpreset "github.com/code-ready/crc/pkg/crc/preset"
+	"github.com/code-ready/crc/pkg/crc/version"
+	"github.com/code-ready/crc/pkg/extract"
+	"github.com/containers/image/v5/copy"
+	"github.com/containers/image/v5/directory"
+	"github.com/containers/image/v5/docker"
+	"github.com/containers/image/v5/signature"
+	"github.com/containers/image/v5/types"
+	v1 "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+const (
+	registryURI = "quay.io"
```

Comment:
can these be defined in a `constants.go` file instead?

Reply:
@gbraad yes we can move it to constants.go, As of now it is here just to have all image related stuff in same package.

Reply:
Only for 2-3 constants which also used for same package I don't feel create a different constants file.